userspace/libsinsp/container.cpp

```diff
@@ -20,6 +20,7 @@ limitations under the License.
 
 #if !defined(MINIMAL_BUILD) && !defined(__EMSCRIPTEN__)
 #include <libsinsp/container_engine/cri.h>
+#include <libsinsp/cri.hpp>
```

Comment:
Mmmh i don't see `get_cri_runtime_type` referenced anywhere in `container.cpp`. Is this the right place to include cri.hpp?
Also, the same

Reply:
I think it's due to `container.cpp` indirectly including `userspace/cri.h`, which has:

```
using cri_interface_v1alpha2 = cri_interface<cri_api_v1alpha2>;
using cri_interface_v1 = cri_interface<cri_api_v1>;
```

When I try to build a Debian package here, libsinsp.so ends up with undefined get_cri_runtime_type symbols:

```
$ nm libsinsp/libsinsp.so | grep get_cri_runtime_type
00000000002e0810 T _ZNK8libsinsp16container_engine3cri20get_cri_runtime_typeEv
                 U _ZNK8libsinsp3cri13cri_interfaceINS0_10cri_api_v1EE20get_cri_runtime_typeEv
                 U _ZNK8libsinsp3cri13cri_interfaceINS0_16cri_api_v1alpha2EE20get_cri_runtime_typeEv
```

Maybe a better option would be to include `libsinsp/cri.hpp` instead of `libsinsp/cri.h` in `userspace/container_engine/cri.h`, or to combine `userspace/cri.h` and `userspace.hpp`?

It's also possible that the [combination of CMake options](https://salsa.debian.org/debian/falcosecurity-libs/-/blob/master/debian/rules) in the Debian rules file is triggering this but I didn't see anything obvious.